app/src/lib/ipc-shared.ts

```diff
@@ -46,6 +46,8 @@ export type RequestChannels = {
   'menu-event': (name: MenuEvent) => void
   log: (level: LogLevel, message: string) => void
   'will-quit': () => void
+  'will-quit-even-updating': () => void
```

Comment:
Is this missing an `If` or something? I'm having a hard time parsing it `will-quit-even-if-updating` or `will-quite-when-updating` perhaps?

Reply:
Probably, sorry, my English sucks 😂